packages/neuron-ui/src/utils/wakeScreen.ts

```diff
@@ -0,0 +1,29 @@
+let wakeLock: WakeLockSentinel | undefined
+
+async function reAwake() {
+  if (document.visibilityState === 'visible') {
+    wakeLock = await navigator.wakeLock.request('screen')
+  } else {
+    wakeLock?.release()
+    wakeLock = undefined
+  }
+}
+
+export async function wakeScreen() {
+  try {
+    if (!wakeLock) {
+      wakeLock = await navigator.wakeLock.request('screen')
+      document.addEventListener('visibilitychange', reAwake)
+    }
+  } catch (err) {
+    console.error('')
+  }
```

Comment:
```suggestion
  if (wakeLock) return

  try {
    wakeLock = await navigator.wakeLock.request('screen')
    document.addEventListener('visibilitychange', reAwake)
  } catch (err) {
    // do nothing here
  }
```

early return for readability